carbon-design-system/carbon-for-ibm-dotcom
# fix(left-nav): fix focus trapping

### Related Ticket(s)
https://jsw.ibm.com/browse/ADCMS-5191

### Description

Fixes the focus trapping on the left-nav component

### Changelog

**Changed**

- Fixes the focus trapping on the left-nav component

### Testing Instructions

- Open the masthead story
- Reduce viewport width to get the mobile version of the nav
- Open the left-nav with the keyboard
- Ensure focus moves to the first item in the left-nav on open
- Ensure focus is trapped in the left-nav, cycling between the close button & last link/button element in both directions.
